docs/FAQ.md

```diff
@@ -133,6 +133,29 @@ The default retention is the same used by K8s. For dynamically provisioned Persi
 
 No. Jiva and cStor volumes are exposed via block storage using iSCSI. Currently only RWO is supported.
 
+## Warning messages during launching PVC
+
+If following warning messages is seen during launching an application, then this messages can be ignored. This messages are seen only during the initial launch of the application pod and is it getting cleared on the next attempt.
```

Comment:
If the following warning messages are displayed when an application is launched, then these messages... These messages...launch of an application pod which is getting cleared on the subsequent attempt.